services/docker-compose.yml

```diff
@@ -236,6 +238,38 @@ services:
       - default
       - interactive_services_subnet
 
+  webserver-garbage-collector:
+    image: ${DOCKER_REGISTRY:-itisfoundation}/webserver:${DOCKER_IMAGE_TAG:-latest}
+    init: true
+    hostname: "{{.Node.Hostname}}-{{.Service.Name}}-{{.Task.Slot}}"
+    environment:
+      <<: *webserver-environment
+      WEBSERVER_GARBAGE_COLLECTOR: '{"GARBAGE_COLLECTOR_INTERVAL_S": 30}'
+    env_file:
+      - ../.env
+      - ../.env-webserver-garbage-collector
+
+    healthcheck:
+      test:
+        [
+          "CMD",
+          "python3",
+          "/home/scu/services/web/server/docker/healthcheck.py",
+          "http://localhost:8080/v0/"
+        ]
+      interval: 30s
+      timeout: 120s
+      retries: 3
+      start_period: 30s
+
+    deploy:
+      placement:
+        constraints:
+          - node.platform.os == linux
```

Comment:
> I feel it would be clearer to just put all the variable without using the *webserver-environment (so, having two times the same thing) and set theme here to null, not in the env-webserver-garbage-collector as it adds another layer in top of 3 another layers.. It would be repeating but easier to understand

I am not sure i understand but i am open to suggestions. Perhaps sometime offline?

Reply:
> Can this service run on the Simcore node ? (in the logic of isolating Simcore services from user nodes). If yes we will need to implement that in the devops part

yes, thisi s part of the stack. it has to work in simcore-node. These placment constratins were there before ...